.github/workflows/unit-integration-tests.yaml

```diff
@@ -0,0 +1,45 @@
+name: Unit and integration tests
+
+on: [pull_request]
+
+jobs:
+  Unittests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+          cache: true
+      - name: Unit test
+        run: ./test/run.sh --unit-only
+      - name: Codecov Unit
+        uses: codecov/codecov-action@v3
+        if: github.event_name == 'pull_request'
+        with:
+          fail_ci_if_error: true
+          files: ./coverage-unit.txt
+          flags: unittests
+          verbose: true
+  Integration:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+          cache: true
+      - name: Integration test
+        run: ./test/integration/run.sh --include-slow
```

Comment:
can we make it to run like that: `./test/run.sh --integration-only`?
so taht `./test/run.sh` would be our source of truth as it goes for running tests